pages/common/docker-network.md

```diff
@@ -0,0 +1,32 @@
+# docker network
+
+> Create and manage docker network.
+> More information: <https://docs.docker.com/engine/reference/commandline/network/>.
+
+- List all available and configured networks on docker daemon:
+
+`docker network ls`
+
+- Create user defined network:
```

Comment:
Why `user defined`?
```suggestion
- Create a network:
```

Reply:
@navarroaxel, `user defined` is because docker comes with a default bridge network as well.

Reply:
Ok @Gau02, then:
```suggestion
- Create a user defined network:
```